src/core/Notification/Notification/Notification.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { Component, forwardRef } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import {
+  HtmlDiv,
+  HtmlDivWithRef,
+  HtmlButton,
+  HtmlButtonProps,
+} from '../../../reset';
+import { Icon } from '../../Icon/Icon';
+import {
+  BaseNotificationProps,
+  notificationClassNames,
+  baseClassName,
+} from '../BaseNotification/BaseNotification';
+import { AutoId } from '../../utils/AutoId/AutoId';
+import { getConditionalAriaProp } from '../../../utils/aria';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../theme';
+import { baseStyles } from './Notification.baseStyles';
+
+export interface NotificationProps extends BaseNotificationProps {
+  /** Text to label the close button. Visible + `aria-label` in regular size and only used as `aria-label` in small screen variant */
+  closeText: string;
+  /** Click handler for the close button */
+  onCloseButtonClick?: () => void;
+  /** Custom props passed to the close button */
+  closeButtonProps?: Omit<HtmlButtonProps, 'onClick' | 'aria-label'>;
+  /** Use small screen styling */
+  smallScreen?: boolean;
+}
+
+interface InnerRef {
+  forwardedRef: React.RefObject<HTMLDivElement>;
+}
+
+class BaseNotification extends Component<NotificationProps & InnerRef> {
+  render() {
+    const {
+      className,
+      ariaLiveMode = 'assertive',
+      status = 'neutral',
+      labelText,
+      children,
+      id,
+      closeText,
+      onCloseButtonClick,
+      smallScreen,
+      closeButtonProps = {},
+      ...passProps
+    } = this.props;
+
+    const {
+      className: customCloseButtonClassName,
+      'aria-describedby': closeButtonPropsAriaDescribedBy,
+      ...closeButtonPassProps
+    } = closeButtonProps;
+    const variantIcon =
+      status === 'neutral'
+        ? 'info'
+        : status === 'success'
+        ? 'checkCircle'
+        : status;
```

Comment:
Not sure what is the best way, but `status` prop seems to have 3 different possibilities: `'neutral' | 'success' | 'error'`.
So, it might be easier for reader if the last option of this condition says it straight instead of defaulting to status, it could be `'error'` as there are not other options left according the interface. 

Or is there some other reason for the current implementation?

Reply:
It seems that this is now fixed for Notification, but **InlineNotification** still has this issue.
If InlineNotification is about to be merged to be "same" as **Notification**, then this obviously is useless change for now.

Reply:
Changed :status to be :'error'